.github/workflows/crmsh-cd.yml

```diff
@@ -20,7 +20,6 @@ jobs:
     uses: ./.github/workflows/crmsh-ci.yml
 
   delivery:
-    if: github.repository == 'ClusterLabs/crmsh' && github.ref_name == 'crmsh-4.6'
```

Comment:
Does `needs: integration` implies this?

Reply:
Yes
```
integration:
    if: github.repository == 'ClusterLabs/crmsh' && github.ref_name == 'crmsh-4.6'
    uses: ./.github/workflows/crmsh-ci.yml
```
integration will run only for crmsh-4.6, then delivery will run after integration